packages/kbn-io-ts-utils/src/non_empty_string_rt/index.test.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { nonEmptyStringRt } from './';
+
+describe('nonEmptyStringRt', () => {
+  it('fails on empty strings', () => {
+    expect(nonEmptyStringRt.decode('')._tag).toEqual('Left');
```

Comment:
nit: using `isRight` feels a bit more solid than comparing against strings and using what seems like an internal identifier (`_tag`)

```suggestion
    expect(isRight(nonEmptyStringRt.decode(''))).toBe(false);
```

where

```
	import { isRight } from 'fp-ts/lib/Either';
```

But doesn't make much of a difference.

Reply:
That's fair, I was being lazy 😄